shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -77,52 +185,272 @@ - (BOOL)canBecomeFirstResponder {
   return YES;
 }
 
+#pragma mark - UITextInput Overrides
+
+- (id<UITextInputTokenizer>)tokenizer {
+  if (_tokenizer == nil) {
+    _tokenizer = [[UITextInputStringTokenizer alloc] initWithTextInput:self];
+  }
+  return _tokenizer;
+}
+
+- (UITextRange*)selectedTextRange {
+  return [[_selectedTextRange retain] autorelease];
```

Comment:
This violates the contract in the interface. `[_selectedTextRange copy]` should fix this.